src/BlockListener.php

```diff
@@ -19,4 +19,31 @@ public function type(): int
     {
         return self::TYPE_BLOCK;
     }
+
+    /**
+     * @param Pick $pick
+     * @return Line
+     * @since 1.3.2
+     */
+    protected function getFirstLine($pick)
```

Comment:
What @lode correctly suggested also also to type hint the input value of the method, this is very important as it ensures the library works the correct way. Please take a look at https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration or take a look at other functions inside the library which does the same.